force-app/main/default/lwc/rd2RecurringDonation/rd2RecurringDonation.js

```diff
@@ -359,6 +355,18 @@ export default class RecurringDonationTable extends LightningElement {
                     return { actions, ...el, nexDonationFormatFirstElement, nexDonationFormatSecondElement };
                 });
             }
+        }).finally(() => {
+          this.data?.forEach((el) => {
```

Comment:
This is pretty hard to follow, why was the pattern changed from using attributes on each 'item' in 'data'? Couldn't you follow similar logic when setting `this.data = data.map((el)...` above?